src/chapters/setup-intro-data-types/setup.rst

```diff
@@ -1,2 +1,218 @@
 Setup For Java
 ===============
+
+For the entirety of this course, we will be coding in Java. We need to not only have
+Java installed on our machines, but also some accompanying technologies to allow
+us to run and edit our Java code.
+
+Java Development Kit
```

Comment:
For each **bold** term, there must be a corresponding index reference.

``.. index:: ! vocabTerm``  The "!" indicates that this is the first reference to the term in the book.